misc/acrn-config/scenario_config/ivshmem_cfg_h.py

```diff
@@ -36,6 +36,13 @@ def write_shmem_regions(config):
             shmem_regions.append((raw_shm_splited[0].strip(), raw_shm_splited[1].strip(), raw_shm_splited[2].strip().split(':')))
             shmem_dev_num += len(raw_shm_splited[2].strip().split(':'))
 
+    if len(shmem_regions) > 0:
+        shmem_cnt = 0
+        print("", file=config)
+        for shmem_region in shmem_regions:
+            print("#define IVSHMEM_SHM_REGION_%d\t"%shmem_cnt, end="", file=config)
+            print('"{}"'.format(shmem_region[0]), file=config)
+            shmem_cnt += 1
```

Comment:
Does there need to be a check that there aren't more than 8 regions specified?

Reply:
Sure. I will update this PR to add sanity check of the max number of shm region is 8 into configuration tool.